Sources_v3/StreamChatUI/Common Views/OnlineIndicatorView.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  OnlineIndicatorView.swift
+//  StreamChatUI
+//
+//  Created by Dominik Bucher on 25/11/2020.
+//  Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+class OnlineIndicatorView: UIView {
+    public var fillColor: UIColor = .green {
+        didSet { layer.borderColor = fillColor.cgColor }
+    }
```

Comment:
Why is it named `fillColor` when it's actually border color??

Reply:
Fixed.